.github/workflows/llvm-build.yml

```diff
@@ -103,7 +104,7 @@ jobs:
         tar czf "${{ env.llvm_install_dir }}.tar.gz" "${{ env.llvm_install_dir }}"
 
     - name: Configure, Build, and Install LLVM (macOS arm64)
-      if: matrix.config.arch == 'arm64' && matrix.config.target-os == 'macos'
+      if: matrix.config.arch == 'arm64' && (matrix.config.target-os == 'ubuntu' || matrix.config.target-os == 'macos')
```

Comment:
The code below CMAKE_OSX_ARCHIECTURES... is it okay to set that for ubuntu linux?

Reply:
Worked fine for me but you are right that it could cause issues, somewhat manually recreated the workflow as a bash file instead of a workflow, was hoping to be able to run the workflow soon.